xml/System.Net.NetworkInformation/IPStatus.xml

```diff
@@ -771,7 +771,7 @@
       </ReturnValue>
```

Comment:
Another nit completely unrelated to your PR: on line 345, there's a reference to RFC 1812; this could also be a link: <https://tools.ietf.org/html/rfc1812>.